code/modules/clothing/spacesuits/rig.dm

```diff
@@ -8,7 +8,7 @@ var/list/all_hardsuit_pieces = list(HARDSUIT_HEADGEAR,HARDSUIT_GLOVES,HARDSUIT_B
 
 //Regular rig suits
 /obj/item/clothing/head/helmet/space/rig
```

Comment:
seems weird to not update the armor stats on the "new" civilian models

our spriters would probably be willing to make you a generic grey rig too

Reply:
there's one in the works currently but not by me fyi